source/h_simulation_settings.rst

```diff
@@ -119,30 +119,47 @@ Function where the ratio between water depth and water level gradient prescribes
 
 *limiter_slope_crossectional_area_2d = 2*
 
-This is a very stable upwind method to redefine the water level depth. It is assumed that the flow behaves as a thin sheet flow. Therefore, the depth is defined as the upwind volume defined by the maximum surface area. 
+The limiter *type 2* is a very stable upstream method, to redefine the water level depth at the cell edge. It is assumed that the flow behaves as a thin sheet flow. Therefore, the depth is defined as the upstream volume devided by the maximum surface area. 
 
 *limiter_slope_crossectional_area_2d = 3, in combination with thin_layer_definition = xx [m]*
 
-In this case the limiter is more or less effective depending of the local depth. In case the depth at the edge base on the down wind water level is larger than the definition that is given of a thin layer, the cross-sectional area is based on the high resolution grid. When this 'down wind' depth is smaller than the thin layer definition, then the limiter described for option 2 is determining the cross-sectional area. In the in between phase the two types of cross-sections are weighed to define a new value.
+The limiter *type 3* provides a smooth transition from the default water depth to the altered one. This transistion depends on the local depth and a user-defined "thin water layer" definition. In case the depth at the edge based on the downstream water level is larger than the thin layer definition, the cross-sectional area is based on the uniform water level assumption. In case the downstream water level is below the thin water layer definition, then limiter 2 determines the cross-sectional area. Finally, if the downstream water level is within the thin water layer depth, these two types of cross-sections are weighed to define the new value (i.e., limiter type 3).
 
-This is decribed in the figure below. Mathematical derivation will follow.
+All the cases are illustrated in the figures below.
 
-.. figure:: image/slopelimiter.png
-   :alt: Limiter for cross-sectional area
-.. figure:: image/lim_slope_3.png
-   :alt: Limiter for cross-sectional area
-   
-   Grid schematisation in a sloping areas. Two alternatives to determine an effective depth for the cross-sectional area. Lower:   The alternatives for the cross-sectional area in case of limiter option 2.
+.. figure:: image/nolimiter.png
+   :figwidth: 1000 px
+   :alt: no_limiter
+
+   Water distribution based on uniform water level assumption showing x-z and x-y profiles
+
+.. figure:: image/limiter1.png
+   :figwidth: 1000 px
+   :alt: limiter_1
+
+   Water distribution based on limiter 1 showing x-z and x-y profiles
+
+.. figure:: image/limiter2.png
+   :figwidth: 1000 px
```

Comment:
Beautifull figures !!!
Can we distribute them in the text as close as possible to the explainations?